libraries/base/java-utils/Utils.java

```diff
@@ -40,6 +40,7 @@
 import eta.runtime.stg.StgContext;
 import eta.runtime.stg.Closure;
 import eta.runtime.io.MemoryManager;
+import eta.runtime.io.ByteArray;
```

Comment:
This import is no longer required right?

Reply:
right